packages/swagger2openapi/index.js

```diff
@@ -307,7 +307,11 @@ function fixupRefs(obj, key, state) {
             }
             else if (inSchema && (options.refSiblings === 'allOf')) {
                 delete obj.$ref;
-                state.parent[state.pkey] = { allOf: [ { $ref: tmpRef }, obj ]};
+                if(typeof obj.description === 'string'){
+                    state.parent[state.pkey] = { allOf: [ { $ref: tmpRef } ], description: obj.description };
+                }else{
```

Comment:
Spaces around `else` please.

Reply:
Your new test case doesn't seem to have an `options.yaml` file, so I can't see how `refSiblings` is set of `allOf` for your test?

Reply:
I have modified the existing issue130-allOf test case by just extending its openapi.yaml, see https://github.com/Mermade/oas-kit/pull/287/files#diff-afd7c7c1bf17445ff8474a91f25cd72cR18